openlibrary/core/db.py

```diff
@@ -165,6 +167,13 @@ def f(*args, **kwargs):
     return f
 
 
+def adapt_datetime_iso(val):
```

Comment:
Nice!  Thanks for doing this.  Let's add three things...
1. The variable name `val` gives the reader the impression that we can provide any value they wish but providing bytes, datetime.date, int, str, etc. will all raise an exception so let's use the variable name `date_time` instead.
2. Let's reinforce that with Python type hint so that the function signature becomes:
    * `def adapt_datetime_iso(date_time: datetime.datetime) -> str:`
 3. Finally, please add one or more [Python doctests ](https://docs.python.org/3/library/doctest.html) to the function's docstring:
```
    """
    Convert datetime.datetime into timezone-naive ISO 8601 datetime string.

    >>> adapt_datetime_iso(datetime.datetime(2023, 4, 5, 6, 7, 8, 9)
    ???
```
After this doctest has been added to the docstring you can run `python3 -m docstring -v openlibrary/core/db.py` on your local machine to see that the test fails.  Then replace ??? with the correct answer and the test should pass.  These tests will be run automatically when new pull requests are added.  If someone breaks this function in a future pull request then the test will fail.
"""